src/cli/markdown.rs

```diff
@@ -11,7 +11,7 @@ struct LineWrapper<'a, T: Terminal> {
     indent: u32,
     margin: u32,
     pos: u32,
-    pub w: &'a mut T,
+    pub(crate) w: &'a mut T,
```

Comment:
this is a public attribute in a private struct. What happens if you just make the attribute non-public?

Reply:
Addressed. Good catch! Thank you.